pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project    xmlns="http://maven.apache.org/POM/4.0.0"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>ch-kafka-email</artifactId>
+    <name>ch-kafka-email</name>
+    <version>unversioned</version>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>companies-house-parent</artifactId>
+        <version>1.0.0-rc2</version>
+    </parent>
+
+    <properties>
+        <!-- JDK -->
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+
+        <!-- Dependency Versions -->
+        <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
+        <ch-kafka.version>1.4.4</ch-kafka.version>
+        <kafka-models.version>1.0.14</kafka-models.version>
+        <jackson.version.core>2.11.2</jackson.version.core>
+        <jackson.version.databind>2.11.2</jackson.version.databind>
+        <spring-framework.version>5.2.8.RELEASE</spring-framework.version>
+
+        <!-- Testing -->
+        <junit.version>4.12</junit.version>
+        <mockito.version>1.10.19</mockito.version>
+
+        <!-- Sonar -->
+        <sonar-maven-plugin.version>3.4.0.905</sonar-maven-plugin.version>
```

Comment:
This is declared in parent pom.xml. Remove duplicate.

Same with other sonar properties. See https://github.com/companieshouse/parent-pom/blob/master/pom.xml#L20